osu.Game.Rulesets.Osu/Mods/OsuModTransform.cs

```diff
@@ -41,16 +67,15 @@ private void applyTransform(DrawableHitObject drawable, ArmedState state)
                     return;
 
                 default:
-                    var hitObject = (OsuHitObject)drawable.HitObject;
-
-                    float appearDistance = (float)(hitObject.TimePreempt - hitObject.TimeFadeIn) / 2;
+                    var h = (OsuHitObject)drawable.HitObject;
+                    float appearDistance = (float)(h.TimePreempt - h.TimeFadeIn) / 2;
 
                     Vector2 originalPosition = drawable.Position;
                     Vector2 appearOffset = new Vector2(MathF.Cos(theta), MathF.Sin(theta)) * appearDistance;
 
                     // the - 1 and + 1 prevents the hit objects to appear in the wrong position.
-                    double appearTime = hitObject.StartTime - hitObject.TimePreempt - 1;
-                    double moveDuration = hitObject.TimePreempt + 1;
+                    double appearTime = h.StartTime - h.TimePreempt - 1;
+                    double moveDuration = h.TimePreempt + 1;
```

Comment:
This was done with good reason (see comment above) but not applied to your new method. Probably should be?

Reply:
I saw it and considered it but the issue I think It tries to address ( hitobjects appearing in the wrong place due to some weird code interactions ?) I didn't experience in my setting so I simply didn't see the need to add it. can do it as a failsafe if you want

Reply:
Please do.